renderer/viewer/three/waypointSprite.ts

```
@@ -240,13 +263,18 @@ export function createWaypointSprite (options: {
     arrowSprite.visible = true
     arrowSprite.position.copy(pos)
 
-    // Angle for rotation relative to screen right/up (derived from camera up vector)
-    const angle = Math.atan2(ry, rx)
-    arrowSprite.material.rotation = angle - Math.PI / 2
+    // Calculate the direction vector from camera to waypoint
+    const direction = new THREE.Vector3(group.position.x, group.position.y, group.position.z).sub(camPos).normalize()
+    
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)